packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -243,7 +268,7 @@ class PlanarFreehandROITool extends ContourSegmentationBaseTool {
            */
           epsilon: 0.1,
         },
-        calculateStats: false,
+        calculateStats: true,
```

Comment:
can we set it back to false, it is brutal to calculate stats on each frame

Reply:
The stats were not being calculated / displayed. Maybe the problem is the tool implementation or when the stats are calculated?